index.html

```diff
@@ -273,14 +327,16 @@ <h1 class="form__h1 poppins">
             <form action="https://formspree.io/f/xpznwqll" method="post">
                 <ul class="form__list">
                     <li class="form__list1">
-                        <input  class="poppins form__list__style" name="name" type="text" id="Firstname" placeholder="Najash" maxlength="30" required />
+                        <input class="poppins form__list__style" name="name" type="text" id="Firstname"
+                            placeholder="Najash" maxlength="30" required />
                     </li>
                     <li>
-                        <input class="poppins form__list__style" type="email" name="email" id="email" placeholder="email@something.com" required />
+                        <input class="poppins form__list__style" type="email" name="email" id="email"
+                            placeholder="email@something.com" required />
                     </li>
                     <li class="form__list2">
-                        <textarea class="poppins form__list__style" name="textarea__form" id="textarea__form" cols="30" rows="10" maxlength="500"
-                            placeholder="Write your message here" required></textarea>
+                        <textarea class="poppins form__list__style" name="textarea__form" id="textarea__form" cols="30"
+                            rows="10" maxlength="500" placeholder="Write your message here" required></textarea>
                     </li>
                     <li>
                         <button type="submit" class="form__button poppins">Get In Touch</button>
```

Comment:
- [ ] The Get In Touch button of the Figma template on the desktop version of the form is aligned in the center. Kindly reposition this button so the form can match with the Figma template.
 -Your design:
<img width="576" alt="image" src="https://user-images.githubusercontent.com/61919617/186716319-ee6444e6-126e-497f-b525-c39661185fee.png">
-Figma template:
<img width="438" alt="image" src="https://user-images.githubusercontent.com/61919617/186716398-3c9da0aa-ac38-4a26-a186-dc3d16e9af90.png">